sim-administration/simmanager/src/integration-test/kotlin/org/ostelco/simcards/admin/SimAdministrationTest.kt

```diff
@@ -199,95 +180,47 @@ class SimAdministrationTest {
     }
 
 
-    @Test
```

Comment:
Is it a good idea to remove the SIM tests?

Reply:
Removed the test because the resource methods were removed. No longer needed.